shared/scales.ts

```diff
@@ -28,5 +28,9 @@ export const SCALES: { [id in ScaleName]: { name: string; values: Array<CardValu
     name: 'Fixed Ratio',
     values: ['1', '2', '4', '8', '16', '32', '64', '128', ...SPECIAL_VALUES_ORDERED],
   },
+  LINEAR_SCALE: {
+      name: 'Linear',
+      values: ['0', '1', '2', '3', '4', '5', ...SPECIAL_VALUES_ORDERED],
```

Comment:
I felt that a linear scale with 6 distinct numbers provides enough differentiation and allows team members to make meaningful distinctions. Additionally, I felt a larger scale, e.g. 1 - 10 offered more differentiation than needed.
This is an anecdotal justification that is based on the SIZES scale that also has 6 distinct values and on the use-cases that I have experienced. A short search did not reveal research related to this and two of the most popular websites with linear estimation for poker planning mentioned varying scales:
1-10: https://www.scrumexpert.com/knowledge/scrum-planning-poker-estimation-scales/
1-5: https://linear.app/docs/estimates#configure